crates/bevy_render/src/render_graph/nodes/pass_node.rs

```diff
@@ -306,6 +313,11 @@ impl DrawState {
         self.index_buffer = Some(buffer);
     }
 
+    pub fn can_draw(&self) -> bool {
+        self.bind_groups.iter().all(|b| b.is_some())
+            && self.vertex_buffers.iter().all(|v| v.is_some())
+    }
+
     pub fn can_draw_indexed(&self) -> bool {
         self.bind_groups.iter().all(|b| b.is_some())
             && self.vertex_buffers.iter().all(|v| v.is_some())
```

Comment:
you can rewrite can_draw_indexed to use can_draw

Reply:
Great idea!